### PR TITLE
Add custom fields to Address

### PR DESCRIPTION
**Pull request by tleguijt**
Implements #230 

- [x] Added a custom field to the Address and AddressDraft.
- [x] Added [Set Shipping Address Custom Type](https://docs.commercetools.com/api/projects/carts#set-shipping-address-custom-type) and [Set Shipping Address Custom Field](https://docs.commercetools.com/api/projects/carts#set-shipping-address-customfield) update actions for carts
- [x] Added [Set Billing Address Custom Type](https://docs.commercetools.com/api/projects/carts#set-billing-address-custom-type) and [Set Billing Address Custom Field](https://docs.commercetools.com/api/projects/carts#set-billing-address-customfield) update actions for carts
- [x] Added [Set Item Shipping Address Custom Type](https://docs.commercetools.com/api/projects/carts#set-itemshipping-address-custom-type) and [Set Item Shipping Address Custom Field](https://docs.commercetools.com/api/projects/carts#set-itemshipping-address-customfield) update actions for carts
- [x] Added [Set Delivery Address Custom Type](https://docs.commercetools.com/api/projects/carts#set-delivery-address-custom-type) and [Set Delivery Address Custom Field](https://docs.commercetools.com/api/projects/carts#set-delivery-address-customfield) update actions for carts
- [x] Added [Set Shipping Address Custom Type](https://docs.commercetools.com/api/projects/orders#set-shipping-address-custom-type) and [Set Shipping Address Custom Field](https://docs.commercetools.com/api/projects/orders#set-shipping-address-customfield) update actions for orders
- [x] Added [Set Billing Address Custom Type](https://docs.commercetools.com/api/projects/orders#set-billing-address-custom-type) and [Set Billing Address Custom Field](https://docs.commercetools.com/api/projects/orders#set-billing-address-customfield) update actions for orders
- [x] Added [Set Item Shipping Address Custom Type](https://docs.commercetools.com/api/projects/orders#set-itemshipping-address-custom-type) and [Set Item Shipping Address Custom Field](https://docs.commercetools.com/api/projects/orders#set-itemshipping-address-customfield) update actions for orders
- [x] Added [Set Delivery Address Custom Type](https://docs.commercetools.com/api/projects/orders#set-delivery-address-custom-type) and [Set Delivery Address Custom Field](https://docs.commercetools.com/api/projects/orders#set-delivery-address-customfield) update actions for orders
- [x] Added [Set Shipping Address Custom Type](https://docs.commercetools.com/api/projects/order-edits#set-shipping-address-custom-type) and [Set Shipping Address Custom Field](https://docs.commercetools.com/api/projects/order-edits#set-shipping-addresscustomfield) update actions for order edits
- [x] Added [Set Billing Address Custom Type](https://docs.commercetools.com/api/projects/order-edits#set-billing-address-custom-type) and [Set Billing Address Custom Field](https://docs.commercetools.com/api/projects/order-edits#set-billing-addresscustomfield) update actions for order edits
- [x] Added [Set Item Shipping Address Custom Type](https://docs.commercetools.com/api/projects/order-edits#set-itemshipping-address-custom-type) and [Set Item Shipping Address Custom Field](https://docs.commercetools.com/api/projects/order-edits#set-itemshipping-addresscustomfield) update actions for order edits
- [x] Added [Set Delivery Address Custom Type](https://docs.commercetools.com/api/projects/order-edits#set-delivery-address-custom-type) and [Set Delivery Address Custom Field](https://docs.commercetools.com/api/projects/order-edits#set-delivery-addresscustomfield) update actions for order edits
- [x] Added [Set Address Custom Type](https://docs.commercetools.com/api/projects/customers#set-customtype-in-address) and [Set Address Custom Field](https://docs.commercetools.com/api/projects/customers#set-customfield-in-address) update actions on customers
- [x] Added [Set Address Custom Type](https://docs.commercetools.com/api/projects/channels#set-customtype-in-address) and [Set Address Custom Field](https://docs.commercetools.com/api/projects/channels#set-customfield-in-address) update actions on channels

### CODE REVIEW

File: api-specs/api/types/cart/updates/CartSetBillingAddressCustomFieldAction.raml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#%RAML 1.0 DataType
+(package): Cart
+(docs-uri): https://docs.commercetools.com/api/projects/carts#set-billing-address-custom-type
```
Comment (jenschude) [inline api-specs/api/types/cart/updates/CartSetBillingAddressCustomFieldAction.raml:3]:
```suggestion
(docs-uri): https://docs.commercetools.com/api/projects/carts#set-billing-address-customfield
```

File: api-specs/api/types/cart/updates/CartSetItemShippingAddressCustomFieldAction.raml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#%RAML 1.0 DataType
+(package): Cart
+(docs-uri): https://docs.commercetools.com/api/projects/carts#set-delivery-address-custom-type
```
Comment (jenschude) [inline api-specs/api/types/cart/updates/CartSetItemShippingAddressCustomFieldAction.raml:3]:
```suggestion
(docs-uri): https://docs.commercetools.com/api/projects/carts#set-itemshipping-address-customfield
```

File: api-specs/api/types/cart/updates/CartSetShippingAddressCustomFieldAction.raml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#%RAML 1.0 DataType
+(package): Cart
+(docs-uri): https://docs.commercetools.com/api/projects/carts#set-shipping-address-custom-type
```
Comment (jenschude) [inline api-specs/api/types/cart/updates/CartSetShippingAddressCustomFieldAction.raml:3]:
```suggestion
(docs-uri): https://docs.commercetools.com/api/projects/carts#set-shipping-address-customfield
```

**Review by jenschude — APPROVED** [review]
small changes

**Comment by tleguijt** [comment]
Great, thanks for the corrections

**Pull request merged**